frontend/my-adopthive-app/src/AdopterList/AdopterList.jsx

```diff
@@ -61,6 +65,34 @@ const AdopterList = () => {
     }
   };
 
+  const handleLike = async (profileId, isAdopter) => {
```

Comment:
similar to above comment, let's try to minimize code duplication across `AdopterList` and `AdopteeList`

Try to think about what is the common behavior that you would want in a shared function first. Then try to think about if there's any slight differences between how the two components would call this function, and use that as input parameters to make a reusable function for both callsites.